Forms/HTMLEditor/FileShortcodeProvider.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace SilverStripe\Forms\HtmlEditor;
```

Comment:
I went with this namespace, as the previous extraction, the EmbedShortcodeProvider, was decided to go into this namespace.

I'm happy to change it, but I'd like to have them all bundled?

My 2 cents on the namespace: `SilverStripe\View\Provider` or `SilverStripe\View\Shortcode` (Or something among those lines). as they are part of the View, not Forms or Assets as such.